batch/sql/add-aggregated-billing-project-resources.sql

```diff
@@ -0,0 +1,127 @@
+CREATE INDEX `batches_billing_project_state` ON `batches` (`billing_project`, `state`);
```

Comment:
I don't see any joins on batches that use this index, am I missing something?

Reply:
It's for the kill switch. See part 2.